src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,11 @@
-export const UserInfo = () => {};
+import { User } from '../../types';
+
+type Props = {
+  user: User | null,
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => (
+  <a className="UserInfo" href={`mailto:${user?.email}`}>
```

Comment:
```suggestion
type Props = {
  user: User,
};

export const UserInfo: React.FC<Props> = ({ user }) => (
  <a className="UserInfo" href={`mailto:${user.email}`}>
```
You render `UserInfo` component only when a user is present